### PR TITLE
Filtering clojures into separate scripts

### PR DESCRIPTION
**Pull request by Sanduhr32**
[guide]: https://github.com/DuncteBot/SkyBot/wiki/Structure-Guide

Pull Request Etiquette
---

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [structure guide][guide].

Fixes # .

- [x] I have tested my changes.
- [ ] I didn't test my changes. I will do that in the following 7 days and agree that this Pull request will be closed if I don't test it in the named time span.
- [x] I know the language good enough for providing high-quality code.

Description
---

Filters clojures into separated scripts to run.

### CODE REVIEW

File: src/main/kotlin/ml/duncte123/skybot/objects/ClojureFilter.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ml.duncte123.skybot.objects
+
+import java.util.regex.Pattern
+
+class ClojureFilter {
```
Comment (duncte123) [inline src/main/kotlin/ml/duncte123/skybot/objects/ClojureFilter.kt]: Missing `@Author` annotation

File: src/main/kotlin/ml/duncte123/skybot/commands/essentials/eval/EvalCommand.kt [hunk 2]
```diff
@@ -229,7 +230,7 @@ class EvalCommand : Command() {
                     if (isRanByBotOwner) {
                         engine.evaluate(script)
                     } else {
-                        protectedShell.evaluate(script)
+                        protectedShell.evaluate(ClojureFilter().filterClojures(script))
```
Comment (duncte123) [inline src/main/kotlin/ml/duncte123/skybot/commands/essentials/eval/EvalCommand.kt]: Can't we store an instance?

Comment (Sanduhr32) [reply src/main/kotlin/ml/duncte123/skybot/commands/essentials/eval/EvalCommand.kt]: sure and it runs recursively

**Comment by Sanduhr32** [comment]
looks recursive to me

**Comment by Sanduhr32** [comment]
Version Bump

**Pull request merged**